Assets/Scripts/Utilities/AudioZone.cs

```diff
@@ -0,0 +1,31 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class AudioZone : MonoBehaviour 
+{
+	[SerializeField] AudioSource sourceData = null;
+	AudioSource playingClip = null;
+
+	void OnTriggerEnter( Collider col )
+	{
+		if ( col.GetComponent<PlayerActor>() )
+		{
+			playingClip = SoundManager.instance.Play3DSoundAndFollow( sourceData, transform );
+		}
+	}
+
+	void OnTriggerExit( Collider col )
```

Comment:
I use col rather than collider because monobehaviour uses collider already. I guess I can use triggerCol, but col seems descriptive enough to me in the context of an OnTrigger function.